test/integration/cel.v2/trainjob_crd_test.go

```diff
@@ -0,0 +1,155 @@
+package cel_v2
+
+import (
+	kubeflowv2 "github.com/kubeflow/training-operator/pkg/apis/kubeflow.org/v2alpha1"
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/kubeflow/training-operator/test/integration/framework"
+)
+
+var _ = ginkgo.Describe("TrainJob CRD", ginkgo.Ordered, func() {
+	var ns *corev1.Namespace
+
+	ginkgo.BeforeAll(func() {
+		fwk = &framework.Framework{}
+		cfg = fwk.Init()
+		ctx, k8sClient = fwk.RunManager(cfg)
+	})
+	ginkgo.AfterAll(func() {
+		fwk.Teardown()
+	})
+
+	ginkgo.BeforeEach(func() {
+		ns = &corev1.Namespace{
+			TypeMeta: metav1.TypeMeta{
+				APIVersion: corev1.SchemeGroupVersion.String(),
+				Kind:       "Namespace",
+			},
+			ObjectMeta: metav1.ObjectMeta{
+				GenerateName: "trainjob-validation-",
+			},
+		}
+		gomega.Expect(k8sClient.Create(ctx, ns)).To(gomega.Succeed())
+	})
+
+	ginkgo.When("TrainJob CR Validation", func() {
+		ginkgo.AfterEach(func() {
+			gomega.Expect(k8sClient.DeleteAllOf(ctx, &kubeflowv2.TrainJob{}, client.InNamespace(ns.Name))).Should(gomega.Succeed())
+		})
+
+		ginkgo.It("Should succeed in creating TrainJob", func() {
+
+			apiGroup := "kubeflow.org"
+			kind := "TrainingRuntime"
```

Comment:
Let's create a new module `constants` under `/pkg/constants/constants.go`, that we will use for common constants like: Kind, APIGroup, etc. We will use them in different places.
WDYT @tenzen-y @akshaychitneni ?

Reply:
Do we want to add these under respective APIs `groupversion_info.go` instead, to make sure the export paths are cleaner while calling these for both the API versions?

Reply:
Sure! I think, the GroupVersion is already set here: https://github.com/kubeflow/training-operator/blob/master/pkg/apis/kubeflow.org/v2alpha1/groupversion_info.go#L29, but not the Kind.
@varshaprasad96 @tenzen-y Where do you think we should put the Kind constants ?

Reply:
It would be good to have `Kind` defined in there too.

Reply:
I have added `Kind` to specific api type files and reused group from groupVersion constant